src/gui/mainwindow.cpp

```diff
@@ -1721,6 +1728,13 @@ void MainWindow::on_actionShowStatusbar_triggered()
     showStatusBar(isVisible);
 }
 
+void MainWindow::on_actionShowFiltersSidebar_triggered()
+{
+    const bool isVisible = static_cast<QAction*>(sender())->isChecked();
```

Comment:
Use `sender()` isn't recommended so please choose another way.

Reply:
Just use `bool checked` parameter.

Reply:
What method is that a parameter of?

Reply:
`QAction::triggered()` signal.

Reply:
When updating the method definition to `MainWindow::on_actionShowFiltersSidebar_triggered(bool checked)` I get this error:

```
linking qbittorrent
moc_mainwindow.o: In function `MainWindow::qt_static_metacall(QObject*, QMetaObject::Call, int, void**) [clone .part.0]':
/root/qBittorrent2/src/moc_mainwindow.cpp:387: undefined reference to `MainWindow::on_actionShowFiltersSidebar_triggered()'
collect2: error: ld returned 1 exit status
Makefile:1049: recipe for target 'qbittorrent' failed
make[1]: *** [qbittorrent] Error 1
make[1]: Leaving directory '/root/qBittorrent2/src'
Makefile:56: recipe for target 'sub-src-install_subtargets' failed
make: *** [sub-src-install_subtargets] Error 2
```

Reply:
>When updating the method definition to MainWindow::on_actionShowFiltersSidebar_triggered(bool checked) I get this error:

We might be able to help you fix it if you push the changes.

Reply:
>We might be able to help you fix it if you push the changes.

Or even leave it as is since all similar methods near it still use the same way. Then we can change them all.